static/css/blog.css

```diff
@@ -497,6 +497,10 @@ img.big-img {
   overflow-x: scroll;
 }
 
+.global-nav {
+  padding: 0;
```

Comment:
Hello @Arhell ,
I looked at the preview and compared the `ul.global-nav` setting for the Documentation versus the Blog.
The change makes the Blog top nav match the other pages from the top nav.
From a quick inspection, the Documentation item picks up the padding from `ul`.
I'd like someone from the Blog sub-project team to look at the change.
LGTM
/cc @kbarnard10